bunfig.toml

```diff
@@ -0,0 +1,4 @@
+[test]
+coverageSkipTestFiles = true
+coverageThreshold = .9
```

Comment:
increased from 0 just for now to test, but will be updating to 1 (100%) after this pr is merged.

Reply:
ended up reverting back to 0, since i saw an issue was opened up specifically for addressing coverage: will resolve there.